dbms/src/AggregateFunctions/QuantileExact.h

```diff
@@ -64,7 +64,7 @@ struct QuantileExact
     }
 
     /// Get the value of the `level` quantile. The level must be between 0 and 1.
-    Value get(Float64 level)
+    virtual Value get(Float64 level)
```

Comment:
Need performance test with many (millions) of aggregate function states.

Reply:
> Need performance test with many (millions) of aggregate function states.

Some of `virtual`'s were redundant in this commit. It's needed only for `getFloat` and `getManyFloat`, so the perfomance of old function won't be affected.